aspnetcore/signalr/java-client.md

```diff
@@ -69,6 +69,13 @@ SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further detail
 
 This can safely be ignored.
 
+## Android Development Notes
+
+With regards to Android SDK compatibility for the SignalR Client features, these items need to be considered when specifying your target Android SDK version:
+
+* The SignalR Java Client will run on **Android API Level 16 and above**.
+* Connecting through the Azure SignalR Service will require Android API Level 21 and above because the [Azure SignalR Service](https://docs.microsoft.com/en-us/azure/azure-signalr/signalr-overview) requires TLS 1.2 and does not support SHA-1-based cipher suites. Android [added support for SHA-256 (and above) cipher suites](https://developer.android.com/reference/javax/net/ssl/SSLSocket) in API Level 20.
```

Comment:
```suggestion
* Connecting through the Azure SignalR Service will require Android API Level 20 and above because the [Azure SignalR Service](https://docs.microsoft.com/en-us/azure/azure-signalr/signalr-overview) requires TLS 1.2 and does not support SHA-1-based cipher suites. Android [added support for SHA-256 (and above) cipher suites](https://developer.android.com/reference/javax/net/ssl/SSLSocket) in API Level 20.
```

I think we can just say level 20 in both places. I originally wrote it this way because we tested on API Level 21, but the support we needed was technically added in 20. API level 20 was only used in Android 4.4W which was only released for Android Wear devices. For phones, the sequence went from 19 straight to 21. That's a complexity that's not really needed :).

Reply:
Did we end up wanting to change this to API level 20?